compiler/src/main/java/com/squareup/anvil/compiler/Utils.kt

```diff
@@ -30,11 +31,13 @@ internal val mergeSubcomponentFqName = FqName(MergeSubcomponent::class.java.cano
 internal val mergeInterfacesFqName = FqName(MergeInterfaces::class.java.canonicalName)
 internal val mergeModulesFqName = FqName(MergeModules::class.java.canonicalName)
 internal val contributesToFqName = FqName(ContributesTo::class.java.canonicalName)
+internal val contributesBindingFqName = FqName(ContributesBinding::class.java.canonicalName)
 internal val daggerComponentFqName = FqName(Component::class.java.canonicalName)
 internal val daggerSubcomponentFqName = FqName(Subcomponent::class.java.canonicalName)
 internal val daggerModuleFqName = FqName(Module::class.java.canonicalName)
 
-internal const val HINT_PACKAGE_PREFIX = "hint.anvil"
+internal const val HINT_CONTRIBUTES_PACKAGE_PREFIX = "hint.anvil"
```

Comment:
I wish I could change this package, but it would be a non-backward compatible change if you depend on library that was compiled with an older Anvil version.